devlab/generate_load.py

```diff
@@ -217,7 +193,8 @@ def _get_body_for_image_creating(_image):
             img = self.glanceclient.images.create(
                 **_get_body_for_image_creating(image))
             img_ids.append(img.id)
-        wait_until_images_created(img_ids[:])
+        self.wait_until_objects_created(img_ids[:], self.check_image_state,
```

Comment:
No, This is correct code, you don't need to modify. In the other places we are not using the list of id after calling self.wait_until_objects_created function.